interp/interp_test.go

```diff
@@ -193,6 +193,9 @@ type concBuffer struct {
 }
 
 func absPath(dir, path string) string {
+	if path == "" {
+		return ""
+	}
```

Comment:
I hadn't noticed we had two copies of this func, one as a method and another as a standalone func. Perhaps move this next to the method, and make the method do `return absPath(r.Dir, path)`.